Block/StaticBlock.php

```diff
@@ -2,53 +2,47 @@
 
 namespace Elgentos\PrismicIO\Block;
 
+use Elgentos\PrismicIO\Exception\ApiNotEnabledException;
 use Elgentos\PrismicIO\Exception\ContextNotFoundException;
 use Elgentos\PrismicIO\Exception\DocumentNotFoundException;
 use Elgentos\PrismicIO\Model\Api;
 use Elgentos\PrismicIO\ViewModel\DocumentResolver;
 use Elgentos\PrismicIO\ViewModel\LinkResolver;
+use Magento\Framework\Exception\NoSuchEntityException;
 use Magento\Framework\View\Element\Context;
+use Psr\Log\LoggerInterface as Logger;
 use stdClass;
 
 class StaticBlock extends AbstractBlock
 {
-    /** @var Api */
-    private $api;
-    /** @var string */
-    private $contentType;
-    /**
-     * @var string|null
-     */
-    private $identifier;
+    private Api $api;
+
+    private Logger $logger;
+
+    private string $contentType;
+
+    private ?string $identifier;
 
-    /**
-     * Constructor.
-     *
-     * @param Context $context
-     * @param DocumentResolver $documentResolver
-     * @param LinkResolver $linkResolver
-     * @param Api $api
-     * @param string $contentType
-     * @param string|null $identifier
-     * @param array $data
-     */
     public function __construct(
         Context $context,
         DocumentResolver $documentResolver,
         LinkResolver $linkResolver,
         Api $api,
+        Logger $logger,
```

Comment:
You added the logger, but I don't see it being used anywhere. Is that correct? And if you do need it, please use the LoggerInterface